src/logging.cc

```diff
@@ -786,7 +794,7 @@ inline void LogDestination::LogToSinks(LogSeverity severity,
                                        size_t message_len) {
   ReaderMutexLock l(&sink_mutex_);
   if (sinks_) {
-    for (int i = sinks_->size() - 1; i >= 0; i--) {
+    for (ptrdiff_t i = sinks_->size() - 1; i >= 0; i--) {
```

Comment:
I'm pretty sure I will get a warning about truncation from size_t (sinks_->size()) to int (i). This was the reason to change the loop in the first place.